charts/nut-exporter/dashboards/default.json

```diff
@@ -0,0 +1,1051 @@
+{
```

Comment:
Instead of maintaining two copies, could we take a copy of the file in the root of the repo at the time the chart is built/published?